draft-ietf-tls-tls13.md

```diff
@@ -3422,16 +3425,25 @@ before destroying the transport.
 Error handling in the TLS Handshake Protocol is very simple. When an
 error is detected, the detecting party sends a message to its
 peer. Upon transmission or receipt of a fatal alert message, both
-parties immediately close the connection.  Whenever an implementation
-encounters a condition which is defined as a fatal alert, it MUST send
-the appropriate alert prior to closing the connection. All alerts defined
-in this section below, as well as all unknown alerts are universally
-considered fatal as of TLS 1.3 (see {{alert-protocol}}).
+parties immediately close the connection.
+
+Whenever an implementation encounters a fatal error condition, it
+SHOULD send an appropriate fatal alert and MUST close the connection
+without sending or receiving any additional data. In the rest of this
+specification, the phrase "{terminate the connection, abort the
+handshake}" is used without a specific alert means that the
+implementation SHOULD send the alert indicated by the descriptions
+below. The phrase "{terminate the connection, abort the handshake}
+with a X alert" MUST send alert X if it sends any alert. All
```

Comment:
I'd add a sentence here stating that any language defining an error response that doesn't use this language is still a MUST in all cases, not a SHOULD, to be clear.